### ISSUE TITLE
Article: Product concept validation

### ISSUE THREAD

**Issue by prokopsimek** [DXHeroes/knowledge-base#142]

**Comment by ridlees** [comment]
What is this about: Product concept validation is the process of gathering evidence around concepts through experimentation to make fast, informed, and de-risked decisions. The usual approaches validate the product against a problem, a market, itself, and against a customer (willingness to pay).
Read more here: https://www.verypossible.com/insights/how-we-work-validating-a-product-concept, https://www.smashingmagazine.com/2016/07/a-lean-approach-to-product-validation/, https://www.viima.com/blog/idea-validation, https://onlinelibrary.wiley.com/doi/10.1002/9781118229248.ch9a
-  Follow our [Contribution guide](https://github.com/DXHeroes/knowledge-base-content/blob/master/CONTRIBUTING.md)
- How to [Git and Pull Requests]( https://github.com/firstcontributions/first-contributions/blob/master/README.md)
- How to [Markdown](https://guides.github.com/features/mastering-markdown/), [Markdown interactive demo](https://www.markdowntutorial.com/lesson/1/) or [Google Docs Addon](https://gsuite.google.com/marketplace/app/docs_to_markdown/700168918607)

**Comment by Sa2003hil** [comment]
I want to work on this issue #142 can you assign this to me.

**Comment by prokopsimek** [comment]
@Sa2003hil It's yours! Please follow the contribution guidelines. Do you have a delivery estimate? Thank you

**Comment by Sa2003hil** [comment]
**@prokopsimek**  Thank you for assigning this issue to me. I've got it!

I'll make sure to follow the contribution guidelines.  

Regarding the delivery estimate, I'll need to take a closer look at the issue and its complexity. I'll get back to you with a more accurate estimate within 2 days. Your patience is greatly appreciated.

Thank you!
Just to confirm, does this task involve creating a Markdown (MD) file?

**Comment by prokopsimek** [comment]
@Sa2003hil You're right. The result is a new PR that contains MD file with the content of your article and probably some image(s) that you are gonna use in the article.
